src/services/catalog.service.ts

```diff
@@ -138,6 +139,9 @@ export class CatalogService {
     let allSubCatOfOther: any = _.get(otherMainCategory, 'subCategories[0]');
 
     _.each(items, (item) => {
+      if (item.vendor) {
+        this.vendors[item.vendor] = true;
```

Comment:
@spadgett, you suggested ```vendors[item.vendor] = true;``` in your last comment :-)  I'm not following what you are suggesting?  ```this.vendors`` needs to be global as that is how we expose the vendors list to other services/controllers.